NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPageGeneral/SettingsPageGeneralSettingsViewController.cs

```diff
@@ -97,22 +99,31 @@ void SetupRadioButtons()
         {
             string appLanguage = LocalPreferencesHelper.GetAppLanguage();
 
-            if (appLanguage == "en")
+            switch (appLanguage)
             {
-                _viewModel.SetSelection(SettingsLanguageSelection.English);
-            }
-            else if (appLanguage == "nn")
-            {
-                _viewModel.SetSelection(SettingsLanguageSelection.Nynorsk);
-            }
-            else
-            {
-                _viewModel.SetSelection(SettingsLanguageSelection.Bokmal);
+                case "en":
+                    _viewModel.SetSelection(SettingsLanguageSelection.English);
```

Comment:
maybe the method as in SettingsLanguageSelectionExtensions so we will remove the switch statement?

Reply:
Done